Atomic/rpm_host_install.py

```diff
@@ -199,10 +165,10 @@ def generate_rpm(name, image_id, labels, exports, destination, values=None, inst
             rpm_root = RPMHostInstall.generate_rpm_from_rootfs(destination, temp_dir, name, image_id, labels, include_containers_file=False, installed_files=installed_files, defaultversion=defaultversion)
             rpm_file = RPMHostInstall.find_rpm(rpm_root)
             if rpm_file:
-                dest_path = os.path.join(destination, "container.rpm")
+                orig_name = "atomic-container-%s.rpm" % name
```

Comment:
nit: I recommend using ``{}`` and ``format`` with string formatting unless there is a need to hold compatibility with very old Python versions.